src/include/log.h

```diff
@@ -351,9 +351,10 @@ struct __wt_log_desc {
                          * MongoDB to detect users accidentally running old binaries on a newer
                          * release. There are no actual log file format changes with version 2 and
                          * 3.
-                         * NOTE: We bumped the log version from 3 to 4 for the same reason.
+                         * NOTE: We bumped the log version from 3 to 4 and from 4 to 5 for the
+                         * same reason.
```

Comment:
Make this consistent with whatever you end up with from the WT-5892 comment I had here.

Reply:
Done.